packages/amplify-category-geo/src/supportedServices.ts

```diff
@@ -0,0 +1,18 @@
+export const supportedServices = {
```

Comment:
no longer used?

Reply:
This is still needed for the `amplify.serviceSelectionPrompt(..)` API and minimal information necessary for that API (service names, aliases, provider) is stored in it.